column_computation.hh

```diff
@@ -62,3 +62,12 @@ public:
     virtual bytes serialize() const override;
     virtual bytes_opt compute_value(const schema& schema, const partition_key& key, const clustering_row& row) const override;
 };
+
+class unsigned_token_column_computation : public column_computation {
```

Comment:
Hm, maybe we should consider renaming the old computation to `legacy_token_column_computation` and name the new one `token_column_computation`. That way, nobody would get tempted to ever use the old, incorrect implementation by mistake. Or, you can simply wrap the old one in `legacy` namespace, which is customary.

Reply:
I agree. But the string name of the computation cannot change, so this will create a somewhat confusing correspondence between the function name and the computation name, so we'll need to make this clear in a comment because nobody will ever remember.

Reply:
> so we'll need to make this clear in a comment because nobody will ever remember.

Added some comments in `column_computation.hh` describing this situation.